docs/source/platform/schema-validation.md

```diff
@@ -98,7 +98,7 @@ type Query {
   getUsers(ids: [ID!]!): [User]!
 
   # what we want to end up with
-  getUsers(ids: [ID!], groupId: ID!): [User]!
+  getUsers(ids: [ID!], groupId: ID): [User]!
```

Comment:
Good catch! I think this might get overridden by https://github.com/apollographql/apollo/pull/336 and #327, but happy to ship this for the mean time :) Thank you for the contribution.